lib/permissions.ts

```diff
@@ -64,3 +64,24 @@ export const canManageCases = (
 
   return false;
 };
+
+export const devCanViewArea = (user: User) => {
```

Comment:
I'd give this a more explicit name, something like: `canUserViewWarningNotes` (the format is basically "can user do this {thing}". In this permissions file `devCanViewArea` is fairly abstract 😅

Reply:
@chrishutchinson 
it's now called canUserViewWarningNotes